com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPassDescriptor.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using UnityEngine.Experimental.Rendering;
+
+namespace UnityEngine.Rendering.Universal
+{
+    internal sealed class ScriptableRenderPassDescriptor
+    {
+        internal int width;
+        internal int height;
+        internal int sampleCount;
+        internal GraphicsFormat[] formats;
+        internal bool isLastPass;
+        internal bool isDepthOnly;
```

Comment:
Considering GraphicsFormat comes from RTHandle. Why not having the other settings in the ScriptableRenderPass itself?

Reply:
SrpDescriptor changes were reverted and now the API is using ConfigureTarget overloads to get this data